interfaces/builtin/usb_raw.go

```diff
@@ -0,0 +1,49 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const usbrawConnectedPlugAppArmor = `
+# Description: Allow raw access to all connected USB devices.
+# Reserved because this gives privileged access to the system.
+# Usage: reserved
+/dev/bus/usb/[0-9][0-9][0-9]/[0-9][0-9][0-9] rw,
+
+# Allow detection of usb devices. Leaks plugged in USB device info
+/sys/bus/usb/devices/ r,
+/sys/devices/pci**/usb[0-9]**/busnum r,
+/sys/devices/pci**/usb[0-9]**/devnum r,
+/sys/devices/pci**/usb[0-9]**/modalias r,
+/sys/devices/pci**/usb[0-9]**/idVendor r,
+/sys/devices/pci**/usb[0-9]**/idProduct r,
+/sys/devices/pci**/usb[0-9]**/descriptors r,
+`
```

Comment:
@teknoraver asked this:
"what about just allowing `/sys/devices/pci**/usb[0-9]** r,`?"

Considering the nature of this interface, please feel free to use `/sys/devices/pci**/usb[0-9]** r,` instead of all these /sys/devices/pci** rules